### PR TITLE
Update Issue Template 

### PR DESCRIPTION
**Pull request by BotBlake**
Changed the Folder Structure from 
`./github/issue_template.md` to `./github/ISSUE_TEMPLATE/issue_template.md`
As per [GitHub Documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms)

Updated the existing Issue Template to have a defining header.
This allows it to appear in the `Template Chooser`.

This is created in response to #956 , so that Issues will from now on use the Template again.

If wanted, I can update the PR to:
- Migrate from Markdown Template to yaml `Issue Form`
- Configure the `Template Chooser` via `config.yml` file (as per [docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser))

The current state of the PR is only to fix what was broken.
However as per GitHub Docs, the old way of having the `issue_template.md` should have worked!
> If you used the legacy workflow to manually create an issue_template.md file in the .github folder and enable blank issues in your config.yml file, the template in issue_template.md will be used when people choose to open a blank issue. If you disable blank issues, the template will never be used.

If you want to check if and how these changes work, feel free to checkout my fork on GH, on which I have set this branch to `default` to have an accurate Preview.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      label: "Kodi Version"
+      description: "Specify the Kodi version you're using."
+      placeholder: "e.g., 20.0"
+      
+  - type: dropdown
+    id: addon_mode
+    attributes:
+      label: "Addon Mode"
+      description: "Select how you're using the addon."
+      options:
```
Comment (oddstr13) [inline .github/ISSUE_TEMPLATE/bug_report.yml:54]: Per the docs, this *shouldn't* give a default, but it does?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      required: false
+
+  - type: textarea
+    id: additional_info
+    attributes:
+      label: "Additional Information"
+      description: "Provide any other relevant information."
+      placeholder: "e.g., Additional context or setup details"
+    validations:
+      required: false
```
Comment (oddstr13) [inline .github/ISSUE_TEMPLATE/bug_report.yml:87]:
```suggestion
      required: false

```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    validations:
+      required: true
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: "Logs"
+      description: "Please provide the logs."
+      placeholder: "Paste your logs here..."
+      render: "bash"
```
Comment (oddstr13) [inline .github/ISSUE_TEMPLATE/bug_report.yml:66]:
```suggestion
      render: "python"
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    validations:
+      required: true
+
+  - type: textarea
+    id: server_logs
+    attributes:
+      label: "Server Logs"
+      description: "Please provide the server logs."
+      placeholder: "Paste your server logs here..."
+      render: "bash"
```
Comment (oddstr13) [inline .github/ISSUE_TEMPLATE/bug_report.yml:76]:
```suggestion
      render: "csharp"
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+      placeholder: "Paste your logs here..."
+      render: "bash"
+    validations:
+      required: true
+
+  - type: textarea
+    id: server_logs
+    attributes:
+      label: "Server Logs"
+      description: "Please provide the server logs."
```
Comment (oddstr13) [inline .github/ISSUE_TEMPLATE/bug_report.yml:74]:
```suggestion
      description: "Please provide server logs (if relevant)."
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+description: "Report an issue with the software."
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: textarea
+    id: summary
+    attributes:
+      label: "Summary"
+      description: "Provide a short description of the issue."
+      placeholder: "e.g., App crashes when opening a file"
```
Comment (oddstr13) [inline .github/ISSUE_TEMPLATE/bug_report.yml:11]:
```suggestion
      placeholder: "When playing a movie via the «Resume from» context menu, I get the error message «xyz»"
```
Placeholders should encourage descriptive responses

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+        - "Direct-path mode ('native')"
+    validations:
+      required: true
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: "Logs"
+      description: "Please provide the logs."
+      placeholder: "Paste your logs here..."
```
Comment (oddstr13) [inline .github/ISSUE_TEMPLATE/bug_report.yml:65]:
```suggestion
      placeholder: "Paste your Kodi logs here..."
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+      render: "bash"
+    validations:
+      required: false
+
+  - type: textarea
+    id: additional_info
+    attributes:
+      label: "Additional Information"
+      description: "Provide any other relevant information."
+      placeholder: "e.g., Additional context or setup details"
```
Comment (oddstr13) [inline .github/ISSUE_TEMPLATE/bug_report.yml:85]:
```suggestion
      placeholder: "Additional context, setup details or screenshots"
```

**Review by oddstr13 — CHANGES_REQUESTED** [review]
I'd rather have dropdowns, with the last option being «Other (please specify)»

None of the dropdowns should have default/pre-selected values, I don't want users to ignore the fields (they did with the old template).

Logs should probably be python code blocks, as python stack traces are expected (and possibly C# for the server logs)

It is unfortunate that there is no upload file option, as it is often preferable to have the log headers which print system info (would essentially duplicate several of the fields, such as OS and version)

**Comment by oddstr13** [comment]
I personally think the «eg.» looks a bit strange (not often used in english?)

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (BotBlake) [reply .github/ISSUE_TEMPLATE/bug_report.yml:87]: I dont understand 🗿

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (BotBlake) [reply .github/ISSUE_TEMPLATE/bug_report.yml:54]: It seems to "start" on the First entry of the DropDown selected.
The only option to "fix" this, would be to have a "Please Select" option in the Dropdown.

However people could then also turn in issues with the "please select" Mode 🗿

**Comment by BotBlake** [comment]
The main issue with Dropdowns is, that there is no Option to do "Other, Please Specify" (at least not to my knowledge)
Thats why I didnt put Dropdowns, as requested, but rather Text fields directly.
I also experimented with having text-fields show up only if "other" was selected, but that also doesnt seem to be supported by GitHub.

However people will not be abled to "ignore" the fields, as they are marked as "required" - so something has to be filled in!

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (oddstr13) [reply .github/ISSUE_TEMPLATE/bug_report.yml:53]: that's fine, I can make CI to tag the issue and yell at them if they do

**Review by oddstr13 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 9]
```diff
@@ -0,0 +1,87 @@
+name: "Bug Report [NEW]"
```
Comment (oddstr13) [inline .github/ISSUE_TEMPLATE/bug_report.yml:1]:
```suggestion
name: "Bug Report"
```

**Pull request merged**